[OSDOCS-20643]: CQA: HCP troubleshooting (1 of 2) (#115371)
openshift/openshift-docs: modules/hcp-ts-rhcos.adoc

:_mod-docs-content-type: PROCEDURE
[id="hcp-ts-rhcos_{context}"]
= Resolving {op-system} image mirroring failures

[role="_abstract"]

> For {hcp} on {VirtProductName} in a disconnected environment, `oc-mirror` fails to automatically mirror the {op-system-first} image to the internal registry.

Reviewer: Is this a better short desc?

```suggestion
For {hcp} on {VirtProductName} in a disconnected environment,  if `oc-mirror` fails to automatically mirror the {op-system-first} image to the internal registry, you can manually mirror the {op-system} image to the internal registry.
```

Author: Yes! Thanks. Good catch.

  | jq -r '.architectures.x86_64.images.kubevirt."digest-ref"'
----

> . Mirror the {op-system} image to your internal registry. Run the following command:

Reviewer: While you are in the file, per [doc guidelines](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#procedure-step-that-introduces-a-command):
```suggestion
. Mirror the {op-system} image to your internal registry by running the following command:
```

spec:
  repositoryDigestMirrors:
    - mirrors:
        - virthost.ostest.test.metalkube.org:5000/localimages/ocp-v4.0-art-dev

>       source: quay.io/openshift-release-dev/ocp-v4.0-art-dev <2>

Reviewer: ```suggestion
      source: quay.io/openshift-release-dev/ocp-v4.0-art-dev
```
